### PR TITLE
gitmoji 1.9.2 (new formula)

### PR DESCRIPTION
**Pull request by maastiff**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/gitmoji.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  sha256 "af7a6c7b0874d2971be909c9c2166b94db4fe1c3d1500f49cfac30d0a5d1b433"
+  depends_on "node"
+
+  def install
+    system "npm", "install", *Language::Node.std_npm_install_args(libexec)
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  test do
+    assert_equal "Gitmojis updated successfully!", shell_output("#{bin}/gitmoji -u").chomp
```
Comment (fxcoudert) [inline Formula/gitmoji.rb]: Homebrew does not include software that autoupdates, as it messes with our own versioning/update system. The software has to be installed in a way that disables its autoupdate capabilities.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/gitmoji.rb (hunk 1 above)
Comment (Maastiff) [reply Formula/gitmoji.rb]: Hi @fxcoudert, Thank you for your feedback. 

Running `gitmoji --update` does not mean updating the software itself but updating the emoji lists cached locally (for offline usage).
A better test should have been to run `gitmoji --list` and check for a particular result but I was a bit worried about stability of this test that would depend on the emoji list expected to evolve.

screenshot 2018-11-03 at 11 28 01

If you have a better way to test in mind, please let me in

Comment (fxcoudert) [reply Formula/gitmoji.rb]: I suggest checking for `🐛 - :bug: - `, which is not likely to change.

Comment (Maastiff) [reply Formula/gitmoji.rb]: Ok, done in https://github.com/Homebrew/homebrew-core/pull/33660/commits/4f7455a6fa56fec21082aa6e19e7170efbe6d376

**Comment by zbeekman** [comment]
@BrewTestBot test this please

**Review by fxcoudert — APPROVED** [review]

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**